meta-spec-0002.md

```diff
@@ -28,18 +28,18 @@ of Projects found in the the `Core Projects` field in the shaded box above.
 This MetaSPEC describes the criteria for being a Core Project and
 explains how to amend the list in the [implementation section](#implementation) below.
 
-These Core Projects play a special role in the SPEC process.
+The Core Projects play a special role in the SPEC process.
 Each Core Project monitors proposed SPECs and often members of their community
 engage in the SPEC design and discussion process.
 Moreover, Core Projects may appear in the `endorsed-by` field of a SPEC header.
 Having a Core project endorse a SPEC signals to the community that the SPEC has
 been vetted by a mature project with a healthy developer review process in-place.
 It also allows more specialized projects to easily see
-whether the Core Projects they depend on have endorse a given SPEC.
+whether the Core Projects they depend on have endorsed a given SPEC.
 
 Since SPECs can be proposed by anyone including members of new or more
 specialized projects, having Core Projects consider these proposals
-provides a convenient mechanism for these projects and individuals
+provides a convenient mechanism for projects and individuals
```

Comment:
I thought this sentence was hard to parse. The "these" is (I think) referring to newer/non-core projects who propose specs. AIUI, the point here is to emphasize that the SPECs are a mechanism for newer projects or users with new ideas to interact with & potentially influence decision in the core projects and thus the ecosystem as a whole.